lib/services/local/desktops.go

```diff
@@ -36,6 +36,28 @@ func NewWindowsDesktopService(backend backend.Backend) *WindowsDesktopService {
 	return &WindowsDesktopService{Backend: backend}
 }
 
+func (s *WindowsDesktopService) GetNonADDesktopsCount(ctx context.Context) (uint64, error) {
```

Comment:
To make this efficient you should also define this in the cache:
- add `GetNonADDesktopsCount` to `windowsDesktopsGetter` in `lib/cache/collections.go`
- implement `GetNonADDesktopsCount` for `*Cache` in `lib/cache/cache.go`, like `GetWindowsDesktops` and `ListWindowsDesktops`
- add `GetNonADDesktopsCount` to the `Cache` interface in `lib/auth/api.go`

Reply:
I've added it to the cache